### ISSUE TITLE
Update to 1.19 fails

### ISSUE THREAD

**Issue by Adrian-Devries** [houseabsolute/DateTime-Locale#23]
I've tried to update Strawberry PERL 5.26.2 to DateTime:.Locale 1.19. Unfortunately this failed.

```cpanm (App::cpanminus) 1.7044 on perl 5.026002 built for MSWin32-x64-multi-thread
Work directory is C:\Users\ADRIAN~1/.cpanm/work/1525585697.6544
You have make C:\Strawberry\c\bin\gmake.exe
You have LWP 6.33
You have C:\gnuWin32\bin\tar.exe, C:\texlive\2018\bin\win32\gzip.exe and C:\ezWinPorts\bin\bzip2.exe
You have C:\texlive\2018\bin\win32\unzip.exe
Searching DateTime::Locale () on cpanmetadb ...
--> Working on DateTime::Locale
Fetching http://www.cpan.org/authors/id/D/DR/DROLSKY/DateTime-Locale-1.19.tar.gz
-> OK
Unpacking DateTime-Locale-1.19.tar.gz
Entering DateTime-Locale-1.19
Checking configure dependencies from META.json
Checking if you have ExtUtils::MakeMaker 6.58 ... Yes (7.34)
Checking if you have Dist::CheckConflicts 0.02 ... Yes (0.11)
Checking if you have File::ShareDir::Install 0.06 ... Yes (0.11)
Configuring DateTime-Locale-1.19
Running Makefile.PL
Checking if your kit is complete...
Looks good
Generating a gmake-style Makefile
Writing Makefile for DateTime::Locale
Writing MYMETA.yml and MYMETA.json
-> OK
Checking dependencies from MYMETA.json ...
Checking if you have File::Temp 0 ... Yes (0.2304)
Checking if you have warnings 0 ... Yes (1.37)
Checking if you have CPAN::Meta::Check 0.011 ... Yes (0.014)
Checking if you have namespace::autoclean 0.19 ... Yes (0.28)
Checking if you have utf8 0 ... Yes (1.19)
Checking if you have IPC::System::Simple 0 ... Yes (1.25)
Checking if you have Test::Warnings 0 ... Yes (0.026)
Checking if you have strict 0 ... Yes (1.11)
Checking if you have ExtUtils::MakeMaker 0 ... Yes (7.34)
Checking if you have Exporter 0 ... Yes (5.72)
Checking if you have Test::More 0.96 ... Yes (1.302136)
Checking if you have Test::Fatal 0 ... Yes (0.014)
Checking if you have Carp 0 ... Yes (1.42)
Checking if you have File::ShareDir 0 ... Yes (1.104)
Checking if you have Specio::Library::String 0 ... Yes (0.42)
Checking if you have File::Spec 0 ... Yes (3.74)
Checking if you have Params::ValidationCompiler 0.13 ... Yes (0.27)
Checking if you have Dist::CheckConflicts 0.02 ... Yes (0.11)
Checking if you have List::Util 1.45 ... Yes (1.50)
Checking if you have Test::Requires 0 ... Yes (0.10)
Checking if you have base 0 ... Yes (2.26)
Checking if you have Specio::Declare 0 ... Yes (0.42)
Checking if you have CPAN::Meta::Requirements 0 ... Yes (2.140)
Checking if you have Test::File::ShareDir::Dist 0 ... Yes (1.001002)
Checking if you have Storable 0 ... Yes (3.11)
Building and testing DateTime-Locale-1.19
cp share\yi-001.pl blib\lib\auto\share\dist\DateTime-Locale\yi-001.pl
(...)
cp lib/DateTime/Locale/zh_Hant_TW.pod blib\lib\DateTime\Locale\zh_Hant_TW.pod
Skip blib\lib\auto\share\dist\DateTime-Locale\es-PA.pl (unchanged)
(...)
Skip blib\lib\auto\share\dist\DateTime-Locale\fr-GA.pl (unchanged)
"C:\Strawberry\perl\bin\perl.exe" "-MExtUtils::Command::MM" "-MTest::Harness" "-e" "undef *Test::Harness::Switches; test_harness(0, 'blib\lib', 'blib\arch')" t/*.t
# 
# Versions for all modules listed in MYMETA.json (including optional ones):
# 
# === Configure Requires ===
# 
#     Module                  Want Have
#     ----------------------- ---- ----
#     Dist::CheckConflicts    0.02 0.11
#     ExtUtils::MakeMaker      any 7.34
#     File::ShareDir::Install 0.06 0.11
# 
# === Build Requires ===
# 
#     Module              Want Have
#     ------------------- ---- ----
#     ExtUtils::MakeMaker  any 7.34
# 
# === Test Requires ===
# 
#     Module                      Want     Have
#     -------------------------- ----- --------
#     CPAN::Meta::Check          0.011    0.014
#     CPAN::Meta::Requirements     any    2.140
#     ExtUtils::MakeMaker          any     7.34
#     File::Spec                   any     3.74
#     File::Temp                   any   0.2304
#     IPC::System::Simple          any     1.25
#     Storable                     any     3.11
#     Test::Fatal                  any    0.014
#     Test::File::ShareDir::Dist   any 1.001002
#     Test::More                  0.96 1.302136
#     Test::Requires               any     0.10
#     Test::Warnings               any    0.026
#     base                         any     2.26
#     utf8                         any     1.19
# 
# === Test Recommends ===
# 
#     Module         Want     Have
#     ---------- -------- --------
#     CPAN::Meta 2.120900 2.150010
# 
# === Runtime Requires ===
# 
#     Module                     Want  Have
#     -------------------------- ---- -----
#     Carp                        any  1.42
#     Dist::CheckConflicts       0.02  0.11
#     Exporter                    any  5.72
#     File::ShareDir              any 1.104
#     List::Util                 1.45  1.50
#     Params::ValidationCompiler 0.13  0.27
#     Specio::Declare             any  0.42
#     Specio::Library::String     any  0.42
#     namespace::autoclean       0.19  0.28
#     strict                      any  1.11
#     warnings                    any  1.37
# 
t/00-report-prereqs.t ........ ok
t/01basic.t .................. ok
t/02iso639-2.t ............... ok
t/03alias.t .................. ok
t/04format-length.t .......... ok
t/05register.t ............... ok
syntax error at -e line 1, at EOF
Execution of -e aborted due to compilation errors.
"C:\Strawberry\perl\bin\perl.exe" unexpectedly returned exit value 255 at t/06storable.t line 54.
# Tests were run but no plan was declared and done_testing() was not seen.
# Looks like your test exited with 255 just after 3.
t/06storable.t ............... 
Dubious, test returned 255 (wstat 65280, 0xff00)
All 3 subtests passed 
t/07script.t ................. ok
t/09version-check.t .......... ok
t/10code-case-insensitive.t .. ok
t/11load-name.t .............. ok
t/zzz-check-breaks.t ......... ok

Test Summary Report
-------------------
t/06storable.t             (Wstat: 65280 Tests: 3 Failed: 0)
  Non-zero exit status: 255
  Parse errors: No plan found in TAP output
Files=12, Tests=810, 30 wallclock secs ( 0.95 usr +  0.06 sys =  1.01 CPU)
Result: FAIL
Failed 1/12 test programs. 0/810 subtests failed.
gmake: *** [Makefile:1645: test_dynamic] Error 255
-> FAIL Installing DateTime::Locale failed. See C:\Users\ADRIAN~1\.cpanm\work\1525585697.6544\build.log for details. Retry with --force to force install it.
```

**Comment by autarch** [comment]
Fixed in 1.20. Thanks!

**Comment by Adrian-Devries** [comment]
Indeed, the issue has been fixed. Thank you!
